utils/utils_test.go

```diff
@@ -0,0 +1,58 @@
+package utils
```

Comment:
This is unit test, E2E tests sit here: https://github.com/dapr/cli/tree/master/tests/e2e

The tests I was asking for are the E2E tests, where the CLI commands is actually executed for real.

Reply:
Yes this PR just for unit test, i'll submit new PR for E2E tests later.

Reply:
Or should i add E2E tests in this PR?